script/releases

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var request = require('request')
+
+var options = {
+  url: 'https://api.github.com/repos/atom/electron/releases',
+  json: true,
+  headers: { 'User-Agent': 'Electron' }
+}
+
+request(options, function (error, response, body) {
+  if (error) return console.log(error)
+  if (!error && response.statusCode == 200) {
+    fs.writeFileSync('_data/releases.json', JSON.stringify(body, null, ' '))
```

Comment:
Super minor, but usually we use `2` as the third param to `JSON.stringify` so that we are always using a 2-space indent, a one space indent here makes `_data/releases.json` look a little strange.